pootle/checks.py

```diff
@@ -262,6 +262,50 @@ def check_settings(app_configs=None, **kwargs):
                     id="pootle.W015",
                 ))
 
+    if settings.POOTLE_TM_SERVER:
+        tm_indexes = []
+
+        if 'local' not in settings.POOTLE_TM_SERVER:
+            errors.append(checks.Critical(
+                _("POOTLE_TM_SERVER['local'] is missing.", server),
+                hint=_("Set POOTLE_TM_SERVER['local']."),
+                id="pootle.C008",
+            ))
+
+        for server in settings.POOTLE_TM_SERVER:
+            if 'INDEX_NAME' not in settings.POOTLE_TM_SERVER[server]:
+                errors.append(checks.Critical(
+                    _("POOTLE_TM_SERVER['%s'] has no INDEX_NAME.", server),
+                    hint=_("Set an INDEX_NAME for POOTLE_TM_SERVER['%s'].",
+                           server),
+                    id="pootle.C009",
+                ))
+            elif settings.POOTLE_TM_SERVER[server]['INDEX_NAME'] in tm_indexes:
+                errors.append(checks.Critical(
```

Comment:
We are telling to have unique indexes to avoid having more than one entry in `POOTLE_TM_SERVERS` to point to the same index and inadvertently push stuff to the wrong TM. For example think of:

```
    {
        'local': {
            'INDEX_NAME': 'translations',
        },
        'whatever': {
            'INDEX_NAME': 'translations',
        },
    }
```

which would allow the `translations` index to be automatically updated, but also allows translations to be pushed from files.